rain_api_core/view_util.py

```diff
@@ -161,6 +154,8 @@ def make_jwt_payload(payload, algo=JWT_ALGO):
         encoded = jwt.encode(payload, get_jwt_keys()['rsa_priv_key'], algorithm=algo)
         log.info(return_timing_object(service="jwt", endpoint="jwt.encode()", duration=duration(timer)))
         return encoded
+        # TODO(reweeden): how can these error types possibly be triggered!? jwt.encode will raise a TypeError on bad
+        # input, but never ValueError or AttributeError. There is also no code here that will raise IndexError
```

Comment:
No idea how any of the code in here would trigger any of the exceptions being caught.

Reply:
lack of a 'rsa_priv_key' from get_jwt_keys() would give us an IndexError, for example.

Reply:
Yea it's my suspicion that it should catching `KeyError` instead.